autofs-5.1.2 - Fix a typo in CREDITS

From: Tomohiro Kusumi <kusumi.tomohiro@gmail.com>

Fix a typo in a contributor's name.
Two other parts of this file says "Aaron".

Signed-off-by: Tomohiro Kusumi <kusumi.tomohiro@gmail.com>
Signed-off-by: Ian Kent <ikent@redhat.com>
---
 CHANGELOG |    1 +
 CREDITS   |    2 +-
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/CHANGELOG b/CHANGELOG
index 9291c20..96260b7 100644
--- a/CHANGELOG
+++ b/CHANGELOG
@@ -6,6 +6,7 @@ xx/xx/2016 autofs-5.1.3
 - drop redundant \n in logerr().
 - Fix size arg of fgets(3).
 - fix libtirpc detection with -Wl,--as-needed.
+- Fix a typo in CREDITS.
 
 15/06/2016 autofs-5.1.2
 =======================
diff --git a/CREDITS b/CREDITS
index 8d5057a..a931de1 100644
--- a/CREDITS
+++ b/CREDITS
@@ -2,7 +2,7 @@
 CREDITS
 =======
 
-Arron Ogden
+Aaron Ogden
 -----------
 
 While other people have contributed to autofs in various ways
